tee-worker/omni-executor/omni-executor.manifest.template

```diff
@@ -75,6 +75,9 @@ loader.env.OE_BSC_TESTNET_URL = { passthrough = true }
 loader.env.OE_ALCHEMY_KEY = { passthrough = true }
 loader.env.OE_PUMPX_SIGNER_URL = { passthrough = true }
 loader.env.OE_PUMPX_WORKER_URL = { passthrough = true }
+loader.env.OE_OMNI_FACTORY_ADDRESS = { passthrough = true }
+loader.env.OE_ENTRY_POINT_ADDRESS = { passthrough = true }
+loader.env.OE_WILDMETA_API_URL = { passthrough = true }
```

Comment:
We need also: `OE_HYPEREVM_TESTNET_URL`, `OE_HYPEREVM_URL`, `OE_ARBITRUM_TESTNET_URL`, `OE_ARBITRUM_URL` 🙏🏼